assets/css/style.css

```diff
@@ -367,4 +367,14 @@ th{
 
 .cmt .viewer .cmtNavBarPanel .attributeinfopanel .panel-heading{
     overflow: auto;
+}
+
+/* fixed size for navbar icons */
+.cmt-widget #cmt .btn-viewer-icn .glyphicon {
+    font-size: 24px;
+}
+
+/* show active tabs | reset opacity to 1 */
+.cmt-widget .show {
+    opacity: 1;
```

Comment:
Makes sure the` .show` class will actually override the` .fade` class. These are used in the tabbed panel overlay. 

In some instances the` .fade` class has this opacity set and it overrides the `.show` class because react-bootstrap changed the working of these classes.